spec/system/admin/admin_manages_pages_spec.rb

```diff
@@ -54,6 +54,18 @@
       end
     end
 
+    it "has client-side validation" do
```

Comment:
This is not the sturdiest of all specs, but testing for HTML validation is a bit tricky, and if this fails we know something changed, which will help find any bugs.